packages/components/src/DataList/hooks/useLayoutMediaQueries.ts

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { BREAKPOINTS, BREAKPOINT_SIZES, Breakpoints } from "../DataList.const";
+
+export function useLayoutMediaQueries() {
```

Comment:
@darryltec the name grid was stuck in my head for quite a while and it slipped in